### PR TITLE
Fix crash

### LINKED ISSUE #703: primer crash
https://github.com/KotlinIsland/basedmypy/pull/699#issuecomment-2206066746

### PR DESCRIPTION
**Pull request by KotlinIsland**
resolves #703

### CODE REVIEW

**Comment by DetachHead** [comment]
add a test?

File: mypy/errorcodes.py [hunk 1]
```diff
@@ -301,6 +301,9 @@ def __hash__(self) -> int:
 POSSIBLE_FUNCTION: Final[ErrorCode] = ErrorCode(
     "possible-function", "possible FunctionType on class", "General", default_enabled=False
 )
+ERM_WHAT: Final[ErrorCode] = ErrorCode(
+    "unhandled-scenario", "don't know what happened here...", "General", default_enabled=False
```
Comment (DetachHead) [inline mypy/errorcodes.py:305]: it should probably say something more descriptive like "an unknown error occurred. raise an issue" or something

Comment (KotlinIsland) [reply mypy/errorcodes.py:305]: to be fair, that text never shows anywhere...

Comment (DetachHead) [reply mypy/errorcodes.py]: ive written error messages before which i also expected to never happen and was often wrong. i think if theres any chance at all the user could see this message it should say that it was some sort of internal/unknown error

**Review by DetachHead — APPROVED** [review]

File: mypy/errorcodes.py (hunk 1 above)
Comment (KotlinIsland) [reply mypy/errorcodes.py]: I mean, this second parameter `description` is never referenced in the code base at all

**Pull request merged**